doc/sphinx/changes.rst

```diff
@@ -14,6 +14,100 @@ Version 8.14
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+Coq version 8.14 integrates many usability improvements, as well
+as an important change in the core language. The main changes include:
+
+  - The :ref:`internal representation <814CaseRepresentation>` of `match` has changed
+    to a more space-efficient and cleaner structure, allowing to fix a completeness
+    issue in presence of cumulative inductive types in the type-checker.
+    The internal representation is now closer to the user-level view of `match`,
+    where the argument context of branches and the inductive binders `in` and `as`
+    do not carry type annotations.
+
+  - A :ref:`new <814CoqNative>` `coqnative` binary performs separate native compilation
+    of libraries, starting from a `vo` file, and is supported by `coq_makefile`.
+    :ref:`Linking of native code libraries <814NativeLinking>` is now performed lazily,
+    bringing a gain of performance on some systems.
+
+  - :ref:`Improvements <814TCCanon>` of typeclasses and canonical structure
+    resolution, allowing more terms to be considered as classes or keys.
+
+  - More control of :ref:`notations <814Notations>` declarations and support
+    for primitive types in string and numeral notations.
+
+  - :ref:`Removal <814Tactics>` of deprecated tactics, notably `omega`, replaced by
+    a greatly improved `lia`, along with many bugfixes.
+
+  - New APIs for interaction with Ltac1, manipulation of inductive types and
+    a Printf module in :ref:`Ltac2 <814Ltac2>`.
+
+  - CoqIDE now integrate an :ref:`Ltac debugger <814CoqIDE>`.
```

Comment:
I suppose you meant
```suggestion
  - CoqIDE now integrates an :ref:`Ltac debugger <814CoqIDE>`.
```
also I must have missed this feature, how does it integrate it exactly?

Reply:
Indeed, it seems to me that most of it was delayed to 8.15. The changes in 8.14 are not even documented AFAIK.

Reply:
In fact, there is a minimalistic changelog entry and the `Set Ltac Debug` does not trigger the (documented) error message anymore. But it isn't clear to me that we should put this forward in the summary highlights, if only because @jfehrle intends to do a preview release of the full-blown Ltac-in-CoqIDE debugger as part of the Coq platform.

Reply:
I would not mention the debugger at all.  The only user-visible bit that made it into 8.14 is allowing access to the existing command-line version of the debugger through the Messages window of CoqIDE.  That was an initial development step, hardly exciting for users.

I expect to put out a preview release of the debugger on top of 8.14 very soon after the initial release of 8.14.  Or, if 8.14 is not out within the next several days, maybe before.

> The changes in 8.14 are not even documented AFAIK.

Nope, they're not.

> full-blown Ltac-in-CoqIDE debugger

I'd say "useful" rather than "full-blown".  Still some interesting features to add.

Reply:
I've removed the point.